docs/containers.conf.5.md

```diff
@@ -774,8 +774,11 @@ this file by creating `/etc/containers/containers.conf` to specify their own
 configuration. Rootless users can further override fields in the config by
 creating a config file stored in the `$HOME/.config/containers/containers.conf` file.
 
-If the `CONTAINERS_CONF` path environment variable is set, just
-this path will be used. This is primarily used for testing.
+If the `CONTAINERS_CONF` path environment variable is set, all system and user
+config files are ignored and only the specified path will be loaded.
+
+If the `CONTAINERS_CONF_OVERRIDE` path environment variable is set, the config
+file will be loaded last even when `CONTAINERS_CONF` is set.
```

Comment:
The word "path", in "path environment variable", very lightly suggests that these could be used in the same sense as `$PATH` and `$LD_LIBRARY_PATH` and other standard envariables that allow a colon-separated list. I would suggest removing the word "path", and also adding an example because I love examples, and also making it super-clear that the value of these envariables must be a **path to a file**, absolute or relative, and the file must exist.

Reply:
Thanks for the feedback!  I will update the docs accordingly :+1: